Site Crashes When DB is Empty or Down

```
2016-08-26T21:27:24.630802+00:00 app[web.1]:     process.nextTick(function() { throw err; });
2016-08-26T21:27:24.630798+00:00 app[web.1]: /app/node_modules/mongodb/lib/utils.js:98
2016-08-26T21:27:24.630810+00:00 app[web.1]:     at /app/node_modules/mongodb-core/lib/connection/pool.js:446:18
2016-08-26T21:27:24.630805+00:00 app[web.1]:     at /app/routes/index.js:50:58
2016-08-26T21:27:24.630803+00:00 app[web.1]:                                   ^
2016-08-26T21:27:24.630809+00:00 app[web.1]:     at /app/node_modules/mongodb-core/lib/cursor.js:579:16
2016-08-26T21:27:24.630804+00:00 app[web.1]: TypeError: Cannot read property 'count' of undefined
2016-08-26T21:27:24.630811+00:00 app[web.1]:     at authenticateStragglers (/app/node_modules/mongodb-core/lib/connection/pool.js:402:16)
2016-08-26T21:27:24.412850+00:00 app[web.1]: GET /dashboard 302 2.810 ms - 82
2016-08-26T21:27:24.630811+00:00 app[web.1]:     at null.messageHandler (/app/node_modules/mongodb-core/lib/connection/pool.js:424:5)
2016-08-26T21:27:24.630807+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb-core/lib/cursor.js:156:5)
2016-08-26T21:27:24.630806+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb/lib/utils.js:96:12)
2016-08-26T21:27:24.630806+00:00 app[web.1]:     at /app/node_modules/mongodb/lib/cursor.js:858:16
2016-08-26T21:27:24.630809+00:00 app[web.1]:     at Object.queryCallback [as cb] (/app/node_modules/mongodb-core/lib/cursor.js:217:18)
2016-08-26T21:27:24.630808+00:00 app[web.1]:     at setCursorNotified (/app/node_modules/mongodb-core/lib/cursor.js:506:3)
2016-08-26T21:27:24.652819+00:00 app[web.1]: 
2016-08-26T21:27:24.664484+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-08-26T21:27:24.664745+00:00 app[web.1]: npm ERR! node v5.11.1
2016-08-26T21:27:24.665627+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2016-08-26T21:27:24.666414+00:00 app[web.1]: npm ERR! Failed at the site-SonarDashboard@1.0.0 start script 'node ./bin/www'.
2016-08-26T21:27:24.665842+00:00 app[web.1]: npm ERR! site-SonarDashboard@1.0.0 start: `node ./bin/www`
2016-08-26T21:27:24.663990+00:00 app[web.1]: npm ERR! Linux 3.13.0-93-generic
2016-08-26T21:27:24.666022+00:00 app[web.1]: npm ERR! Exit status 1
2016-08-26T21:27:24.667682+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2016-08-26T21:27:24.666214+00:00 app[web.1]: npm ERR! 
2016-08-26T21:27:24.666605+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-08-26T21:27:24.667073+00:00 app[web.1]: npm ERR! not with npm itself.
2016-08-26T21:27:24.668273+00:00 app[web.1]: npm ERR!     npm owner ls site-SonarDashboard
2016-08-26T21:27:24.667906+00:00 app[web.1]: npm ERR!     npm bugs site-SonarDashboard
2016-08-26T21:27:24.667283+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2016-08-26T21:27:24.667479+00:00 app[web.1]: npm ERR!     node ./bin/www
2016-08-26T21:27:24.675090+00:00 app[web.1]: 
2016-08-26T21:27:24.668070+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2016-08-26T21:27:24.666807+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the site-SonarDashboard package,
2016-08-26T21:27:24.665266+00:00 app[web.1]: npm ERR! npm  v3.8.6
2016-08-26T21:27:24.675393+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2016-08-26T21:27:24.675611+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2016-08-26T21:27:24.668481+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2016-08-26T21:27:24.774622+00:00 heroku[web.1]: Process exited with status 1
2016-08-26T21:27:24.644253+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/networksbyloadtime" host=www.site-sonar.com request_id=1144baef-65f8-463f-8c6e-dd9997c7b203 fwd="71.91.201.231" dyno=web.1 connect=1ms service=180ms status=503 bytes=0
2016-08-26T21:27:24.786858+00:00 heroku[web.1]: State changed from up to crashed
2016-08-26T21:27:24.787682+00:00 heroku[web.1]: State changed from crashed to starting
2016-08-26T21:27:26.038937+00:00 heroku[web.1]: Starting process with command `npm start`
2016-08-26T21:27:27.993223+00:00 app[web.1]: > site-SonarDashboard@1.0.0 start /app
2016-08-26T21:27:27.993224+00:00 app[web.1]: > node ./bin/www
2016-08-26T21:27:27.993225+00:00 app[web.1]: 
2016-08-26T21:27:27.993207+00:00 app[web.1]: 
2016-08-26T21:27:28.475227+00:00 app[web.1]: Fri, 26 Aug 2016 21:27:28 GMT body-parser deprecated bodyParser: use individual json/urlencoded middlewares at app.js:24:9
2016-08-26T21:27:28.493602+00:00 app[web.1]: Fri, 26 Aug 2016 21:27:28 GMT body-parser deprecated undefined extended: provide extended option at node_modules/body-parser/index.js:105:29
2016-08-26T21:27:28.921920+00:00 heroku[web.1]: State changed from starting to up
2016-08-26T21:27:29.600325+00:00 app[web.1]: /app/node_modules/mongodb/lib/utils.js:98
2016-08-26T21:27:29.600334+00:00 app[web.1]:     process.nextTick(function() { throw err; });
2016-08-26T21:27:29.600334+00:00 app[web.1]:                                   ^
2016-08-26T21:27:29.600337+00:00 app[web.1]: 
2016-08-26T21:27:29.600338+00:00 app[web.1]:     at /app/routes/index.js:50:58
2016-08-26T21:27:29.600339+00:00 app[web.1]:     at /app/node_modules/mongodb/lib/cursor.js:858:16
2016-08-26T21:27:29.600339+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb/lib/utils.js:96:12)
2016-08-26T21:27:29.600338+00:00 app[web.1]: TypeError: Cannot read property 'count' of undefined
2016-08-26T21:27:29.600340+00:00 app[web.1]:     at handleCallback (/app/node_modules/mongodb-core/lib/cursor.js:156:5)
2016-08-26T21:27:29.600340+00:00 app[web.1]:     at setCursorNotified (/app/node_modules/mongodb-core/lib/cursor.js:506:3)
2016-08-26T21:27:29.600341+00:00 app[web.1]:     at /app/node_modules/mongodb-core/lib/cursor.js:579:16
2016-08-26T21:27:29.600342+00:00 app[web.1]:     at Object.queryCallback [as cb] (/app/node_modules/mongodb-core/lib/cursor.js:217:18)
2016-08-26T21:27:29.600342+00:00 app[web.1]:     at /app/node_modules/mongodb-core/lib/connection/pool.js:446:18
2016-08-26T21:27:29.600343+00:00 app[web.1]:     at authenticateStragglers (/app/node_modules/mongodb-core/lib/connection/pool.js:402:16)
2016-08-26T21:27:29.600343+00:00 app[web.1]:     at null.messageHandler (/app/node_modules/mongodb-core/lib/connection/pool.js:424:5)
2016-08-26T21:27:29.610124+00:00 app[web.1]: 
2016-08-26T21:27:29.617662+00:00 app[web.1]: npm ERR! Linux 3.13.0-91-generic
2016-08-26T21:27:29.617981+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "start"
2016-08-26T21:27:29.618181+00:00 app[web.1]: npm ERR! node v5.11.1
2016-08-26T21:27:29.618618+00:00 app[web.1]: npm ERR! npm  v3.8.6
2016-08-26T21:27:29.618871+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2016-08-26T21:27:29.618997+00:00 app[web.1]: npm ERR! site-SonarDashboard@1.0.0 start: `node ./bin/www`
2016-08-26T21:27:29.619110+00:00 app[web.1]: npm ERR! Exit status 1
2016-08-26T21:27:29.619274+00:00 app[web.1]: npm ERR! 
2016-08-26T21:27:29.619434+00:00 app[web.1]: npm ERR! Failed at the site-SonarDashboard@1.0.0 start script 'node ./bin/www'.
2016-08-26T21:27:29.619603+00:00 app[web.1]: npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-08-26T21:27:29.619760+00:00 app[web.1]: npm ERR! If you do, this is most likely a problem with the site-SonarDashboard package,
2016-08-26T21:27:29.619915+00:00 app[web.1]: npm ERR! not with npm itself.
2016-08-26T21:27:29.620070+00:00 app[web.1]: npm ERR! Tell the author that this fails on your system:
2016-08-26T21:27:29.620240+00:00 app[web.1]: npm ERR!     node ./bin/www
2016-08-26T21:27:29.620394+00:00 app[web.1]: npm ERR! You can get information on how to open an issue for this project with:
2016-08-26T21:27:29.620552+00:00 app[web.1]: npm ERR!     npm bugs site-SonarDashboard
2016-08-26T21:27:29.620706+00:00 app[web.1]: npm ERR! Or if that isn't available, you can get their info via:
2016-08-26T21:27:29.620861+00:00 app[web.1]: npm ERR!     npm owner ls site-SonarDashboard
2016-08-26T21:27:29.621015+00:00 app[web.1]: npm ERR! There is likely additional logging output above.
2016-08-26T21:27:29.624284+00:00 app[web.1]: 
2016-08-26T21:27:29.624499+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2016-08-26T21:27:29.624655+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2016-08-26T21:27:29.720569+00:00 heroku[web.1]: State changed from up to crashed
2016-08-26T21:27:29.604169+00:00 heroku[router]: at=error code=H13 desc="Connection closed without response" method=GET path="/networksbyloadtime" host=www.site-sonar.com request_id=fb311403-d8d1-4c26-9253-1f77c569e0eb fwd="71.91.201.231" dyno=web.1 connect=0ms service=224ms status=503 bytes=0
2016-08-26T21:27:29.679117+00:00 heroku[web.1]: Process exited with status 1
2016-08-26T21:27:30.005601+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/networksbyloadtime" host=www.site-sonar.com request_id=54044eb8-64b7-47e2-8fa0-3f7318d7e76d fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:32.724499+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=45792c33-c1f2-4904-b922-13c912f52d2d fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:33.708415+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=61b214f4-a977-4c12-a869-ff9f5b4e72e8 fwd="72.222.254.2" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:34.667307+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=81380d20-a7df-4b84-8982-e0bc95ac98f9 fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:36.086353+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=4bccc3c2-a0a9-45d0-9fa5-fc0477b311f5 fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:36.380616+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=ad5f23ee-59e8-4c58-a765-05baac723fdc fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:27:37.006309+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=bfbaf21e-5da0-4243-abdd-e40f2e76a527 fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:28:11.020726+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=2a45f0cb-5410-4659-a32f-b2feb9560720 fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:28:12.306674+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=www.site-sonar.com request_id=e2ec149d-003a-454e-81b4-ba59a7dd11e6 fwd="71.91.201.231" dyno= connect= service= status=503 bytes=
2016-08-26T21:28:16.426725+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=POST path="/log/" host=ultra-lightbeam.herokuapp.com request_id=00be13af-8c12-483a-bf40-9e95e59acd18 fwd="199.73.114.67" dyno= connect= service= status=503 bytes=
```

--- a/routes/index.js
+++ b/routes/index.js
@@ -32,6 +32,12 @@ router.get('/networksbyloadtime', function(req,res) {
     flip_nblt = sort;
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -45,9 +51,15 @@ router.get('/networksbyloadtime', function(req,res) {
                         $sort: { count: 1 }
                     }
                     ]).toArray( function (err, averageCount){
+                        if(err!=null){
+
+                            console.log(err);
+                            res.render("error.html");
+                            return;
+                        }
                         var size = averageCount.length;
                         var median = Math.floor(size/4);
-                        var filter = averageCount[median].count;
+                        var filter = averageCount[median]!=null? averageCount[median].count : 0;
 
                         benchmarkDB.aggregate([
                             {
@@ -109,6 +121,12 @@ router.get('/networksbyloadtime', function(req,res) {
                                 $limit: 100
                             }
                         ]).toArray(function (err, avgLoadTimes){
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             benchmarkDB.find().count(function (err, total) {
                                 res.render('networksbyloadtime.html', { records : avgLoadTimes});
                                 db.close();
@@ -134,6 +152,12 @@ router.get('/sitesbyloadtime', function(req,res) {
     flip_sblt = sort;
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -147,9 +171,15 @@ router.get('/sitesbyloadtime', function(req,res) {
                         $sort: { count: 1 }
                     }
                 ]).toArray( function (err, averageCount){
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     var size = averageCount.length;
                     var median = Math.floor(size/4);
-                    var filter = averageCount[median].count;
+                    var filter = averageCount[median]!=null? averageCount[median].count : 0;
 
                     benchmarkDB.aggregate([
                         {
@@ -211,7 +241,19 @@ router.get('/sitesbyloadtime', function(req,res) {
                             $limit: 100
                         }
                     ]).toArray(function (err, avgLoadTimes){
+                        if(err!=null){
+
+                            console.log(err);
+                            res.render("error.html");
+                            return;
+                        }
                         benchmarkDB.find().count(function (err, total) {
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             res.render('sitesbyloadtime.html', { records : avgLoadTimes});
                             db.close();
                         });
@@ -235,6 +277,12 @@ router.get('/sitesbyfilesize', function(req,res) {
     flip_sbfs = sort;
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -248,9 +296,15 @@ router.get('/sitesbyfilesize', function(req,res) {
                         $sort: { count: 1 }
                     }
                 ]).toArray( function (err, averageCount) {
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     var size = averageCount.length;
                     var median = Math.floor(size / 4);
-                    var filter = averageCount[median].count;
+                    var filter = averageCount[median]!=null? averageCount[median].count : 0;
 
                     benchmarkDB.aggregate([
                         {
@@ -320,7 +374,19 @@ router.get('/sitesbyfilesize', function(req,res) {
                             $limit: 100
                         }
                     ]).toArray(function (err, avgLoadTimes) {
+                        if(err!=null){
+
+                            console.log(err);
+                            res.render("error.html");
+                            return;
+                        }
                         benchmarkDB.find().count(function (err, total) {
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             res.render('sitesbyfilesize.html', {records: avgLoadTimes});
                             db.close();
                         });
@@ -345,6 +411,12 @@ router.get('/networksbyfilesize', function(req,res) {
     flip_nbfs = sort;
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -358,9 +430,15 @@ router.get('/networksbyfilesize', function(req,res) {
                         $sort: { count: 1 }
                     }
                 ]).toArray( function (err, averageCount) {
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     var size = averageCount.length;
                     var median = Math.floor(size / 4);
-                    var filter = averageCount[median].count;
+                    var filter = averageCount[median]!=null? averageCount[median].count : 0;
 
                     benchmarkDB.aggregate([
                         {
@@ -427,7 +505,19 @@ router.get('/networksbyfilesize', function(req,res) {
                             $limit: 100
                         }
                     ]).toArray(function (err, fileSizes){
+                        if(err!=null){
+
+                            console.log(err);
+                            res.render("error.html");
+                            return;
+                        }
                         benchmarkDB.find().count(function (err, total) {
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             res.render('networksbyfilesize.html', { records : fileSizes});
                             db.close();
                         });
@@ -448,6 +538,12 @@ router.get('/networkstats', function(req, res){
 
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 /* Histogram stats*/
@@ -466,6 +562,12 @@ router.get('/networkstats', function(req, res){
                         }
                     }
                 ]).toArray(function (err, recordsForNetwork){
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     /* Asset type classification*/
                         benchmarkDB.aggregate([
                             {
@@ -515,6 +617,12 @@ router.get('/networkstats', function(req, res){
                                 $sort : { count : -1 }
                             }
                         ]).toArray(function (err, assetTypes){
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             /* Top level stats*/
                             benchmarkDB.aggregate([
                                 {
@@ -564,6 +672,12 @@ router.get('/networkstats', function(req, res){
                                 }
 
                             ]).toArray(function(err,overall ){
+                                if(err!=null){
+
+                                    console.log(err);
+                                    res.render("error.html");
+                                    return;
+                                }
                                 res.render('networkstats.html', {
                                     overall: overall,
                                     records : recordsForNetwork,
@@ -592,6 +706,12 @@ router.get('/sitestats', function(req, res){
     var site = req.query.site;
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -609,6 +729,12 @@ router.get('/sitestats', function(req, res){
                         }
                     }
                 ]).toArray(function (err, recordsForSite){
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     benchmarkDB.aggregate([
                         {
                             $match: {
@@ -657,6 +783,12 @@ router.get('/sitestats', function(req, res){
                             $sort : { count : -1 }
                         }
                     ]).toArray(function (err, assetTypes) {
+                        if(err!=null){
+
+                            console.log(err);
+                            res.render("error.html");
+                            return;
+                        }
                         /* Top level stats*/
                         benchmarkDB.aggregate([
                             {
@@ -708,6 +840,12 @@ router.get('/sitestats', function(req, res){
                                 $limit : 5
                             }
                         ]).toArray(function (err, networkWiseStats) {
+                            if(err!=null){
+
+                                console.log(err);
+                                res.render("error.html");
+                                return;
+                            }
                             benchmarkDB.aggregate([
                                 {
                                     $match: {
@@ -756,6 +894,12 @@ router.get('/sitestats', function(req, res){
                                 }
 
                             ]).toArray(function (err, overall){
+                                if(err!=null){
+
+                                    console.log(err);
+                                    res.render("error.html");
+                                    return;
+                                }
                                 res.render('sitestats.html', {
                                     overall : overall, //Top level stats for site
                                     networkWiseStats: networkWiseStats, // Network level stats
@@ -801,6 +945,12 @@ router.get('/search', function(req, res){
 
     try {
         MongoClient.connect(url, function(err, db) {
+            if(err!=null){
+
+                console.log(err);
+                res.render("error.html");
+                return;
+            }
             var benchmarkDB = db.collection('benchmark_logs');
             try {
                 benchmarkDB.aggregate([
@@ -816,6 +966,12 @@ router.get('/search', function(req, res){
                         }
                     }
                 ]).toArray(function(err,searchResults){
+                    if(err!=null){
+
+                        console.log(err);
+                        res.render("error.html");
+                        return;
+                    }
                     res.render("search.html", {
                         searchResults :searchResults,
                         category: searchBy,

--- a/routes/index.js
+++ b/routes/index.js
@@ -73,28 +73,28 @@ router.get('/networksbyloadtime', function(req,res) {
                                     {
                                         $divide:[
                                             {$subtract:[
-                                                {$multiply:['$avgLoadTime',1000]},
-                                                {$mod:[{$multiply:['$avgLoadTime',1000]}, 1]}
+                                                {$multiply:['$avgLoadTime',10]},
+                                                {$mod:[{$multiply:['$avgLoadTime',10]}, 1]}
                                             ]},
-                                            1000
+                                            10
                                         ]
                                     },
                                     low:{
                                         $divide:[
                                             {$subtract:[
-                                                {$multiply:['$low',1000]},
-                                                {$mod:[{$multiply:['$low',1000]}, 1]}
+                                                {$multiply:['$low',10]},
+                                                {$mod:[{$multiply:['$low',10]}, 1]}
                                             ]},
-                                            1000
+                                            10
                                         ]
                                     },
                                     high:{
                                         $divide:[
                                             {$subtract:[
-                                                {$multiply:['$high',1000]},
-                                                {$mod:[{$multiply:['$high',1000]}, 1]}
+                                                {$multiply:['$high',10]},
+                                                {$mod:[{$multiply:['$high',10]}, 1]}
                                             ]},
-                                            1000
+                                            10
                                         ]
                                     },
                                     count: "$count"
@@ -175,28 +175,28 @@ router.get('/sitesbyloadtime', function(req,res) {
                                 {
                                     $divide:[
                                         {$subtract:[
-                                            {$multiply:['$avgLoadTime',1000]},
-                                            {$mod:[{$multiply:['$avgLoadTime',1000]}, 1]}
+                                            {$multiply:['$avgLoadTime',10]},
+                                            {$mod:[{$multiply:['$avgLoadTime',10]}, 1]}
                                         ]},
-                                        1000
+                                        10
                                     ]
                                 },
                                 low:{
                                     $divide:[
                                         {$subtract:[
-                                            {$multiply:['$low',1000]},
-                                            {$mod:[{$multiply:['$low',1000]}, 1]}
+                                            {$multiply:['$low',10]},
+                                            {$mod:[{$multiply:['$low',10]}, 1]}
                                         ]},
-                                        1000
+                                        10
                                     ]
                                 },
                                 high:{
                                     $divide:[
                                         {$subtract:[
-                                            {$multiply:['$high',1000]},
-                                            {$mod:[{$multiply:['$high',1000]}, 1]}
+                                            {$multiply:['$high',10]},
+                                            {$mod:[{$multiply:['$high',10]}, 1]}
                                         ]},
-                                        1000
+                                        10
                                     ]
                                 },
                                 count: "$count"

--- a/views/error.html
+++ b/views/error.html
@@ -1,0 +1,10 @@
+{% extends "layout.html" %}
+{% block body %}
+
+<div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+    <p> The dashboard is currently unavailable. Thank you for your patience.</p>
+</div>
+{% endblock %}
+
+{% block scripts %}
+{% endblock %}

